package.json

```diff
@@ -22,9 +22,9 @@
     "@react-native-async-storage/async-storage": "^1.17.7",
     "@react-native-firebase/app": "^16.5.0",
     "@react-native-firebase/messaging": "^16.5.0",
-    "@react-navigation/bottom-tabs": "^5.11.15",
-    "@react-navigation/native": "^5.9.8",
-    "@react-navigation/stack": "^5.14.9",
+    "@react-navigation/bottom-tabs": "^6.x.x",
+    "@react-navigation/native": "^6.x.x",
+    "@react-navigation/stack": "^6.x.x",
```

Comment:
Good catch. I thought the `npm install` command was supposed to specify a version instead of leaving the `x`s. Fixed on [ded34e4](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/306/commits/ded34e420f779b3c6157a8fb79807cf7b46904fc)